### DISCUSSION TITLE
Do you know if there are some 2D brain datasets which are appropriate to do registration?

### DISCUSSION THREAD

**Discussion by YMBrianHung** [adalca/medical-datasets#5] [Q&A] [answered]
Do you know if there are some 2D brain datasets which are appropriate to do registration?

**Comment by adalca** [comment, marked as answer]
well, most brain imaging I know is 3D -- but we've actually just processed the OASIS dataset in a way that might help you -- after affine registration to talairach, we extracted the middle 2D slice and prepared anatomical annotations appropriate for that slice as well. I'll try to add the link and resources to this website in the next day or so.

**Comment by adalca** [comment]
Yes, it's standard to do affine normalization before deformable registration in 3D.

I am not too familiar with the LPBA40 dataset, so I'm not sure of this second version from that description only. However, doing affine registration should be fairly straightforward on standard images/datasets.

**Comment by adalca** [comment]
I've added the OASIS data here, which includes a 2D (midslice) download - https://github.com/adalca/medical-datasets/blob/master/neurite-oasis.md

**Comment by adalca** [comment]
Yes they are affinely registered, this should all be in the readme. I can't really think of a brain 2D data set where there will be affine differences, but you could take the data set we released and apply random affine differences to the images and try to recover them.


What is the actual problem you are trying to solve? Do you have a real-world data set you are trying to solve?

--
HMS/MIT

typed on Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by adalca** [comment]
Moving this to discussions
